### DISCUSSION TITLE
Automated web login inside a Github action fails because the device is not authorised

### DISCUSSION THREAD

**Discussion by mdelapenya** [community/community#38654] [Other Feature Feedback, Questions, & Ideas]
Hi 👋  I'm having an issue while automating a Cypress test that needs to login on Github using the web UI. The login happens but at the moment the credentials are submitted in the web form, the test fails in the Github action and I receive an email with the following:

> A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.
>
>Device: Chrome on Linux
>Verification code: ******
>
>If you did not attempt to sign in to your account, your password may be compromised. Visit https://github.com/settings/security to create a new, strong password for your GitHub account.
>
>If you'd like to automatically verify devices in the future, consider enabling two-factor authentication on your account. Visit https://docs.github.com/articles/configuring-two-factor-authentication to learn about two-factor authentication.
>
>If you decide to enable two-factor authentication, ensure you retain access to one or more account recovery methods. See https://docs.github.com/articles/configuring-two-factor-authentication-recovery-methods in the GitHub Help.
>
>Thanks,
>The GitHub Team

The test pass locally because my machine is authorized, but I wonder how I can authorize a Github runner. I've read all those above links and cannot find a way to automate the login. Is this use case allowed under the context of a Github action?

**Comment by jkarni** [comment]
Have you find a way of doing this? I'm having the same issue.
